django/website/appconf/migrations/0004_auto_20160621_1506.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('appconf', '0003_add_logframe_to_settings'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='settings',
+            name='logframe',
+            field=models.OneToOneField(to='logframe.LogFrame'),
+        ),
+    ]
```

Comment:
All settings should be bound to a logframe, but, to begin with, there is a settings instance that isn't connected to a logframe. To deal with this, the one to one field binding settings to logframes is initialised with a null value. The settings object is then associated with the first logframe in the database. Once this is done, the ability to have settings that aren't bound to a logframe is removed.